diff --git a/src/rules/node/file-extension-in-import.d.ts b/src/rules/node/file-extension-in-import.d.ts
index 5cd714b..d563a34 100644
--- a/src/rules/node/file-extension-in-import.d.ts
+++ b/src/rules/node/file-extension-in-import.d.ts
@@ -12,7 +12,7 @@ export interface FileExtensionInImportRule {
       'always' | 'never',
       {
         tryExtensions?: string[];
-        [k: string]: 'always' | 'never';
+        [ext: `.${string}`]: 'always' | 'never';
       },
     ]
   >;
