diff --git a/src/rules/unicorn/import-index.d.ts b/src/rules/unicorn/import-index.d.ts
index 513d1ca..301f7d4 100644
--- a/src/rules/unicorn/import-index.d.ts
+++ b/src/rules/unicorn/import-index.d.ts
@@ -7,5 +7,15 @@ export interface ImportIndexRule {
    *
    * @see [import-index](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v46.0.0/docs/deprecated-rules.md#import-index)
    */
-  'unicorn/import-index': Rule<[RuleLevel]>;
+  'unicorn/import-index': Rule<
+    [
+      RuleLevel,
+      {
+        /**
+         * @deprecated
+         */
+        ignoreImports?: boolean;
+      },
+    ]
+  >;
 }
