diff --git a/src/rules/unicorn/prefer-node-protocol.d.ts b/src/rules/unicorn/prefer-node-protocol.d.ts
index 8da938a..a1bfede 100644
--- a/src/rules/unicorn/prefer-node-protocol.d.ts
+++ b/src/rules/unicorn/prefer-node-protocol.d.ts
@@ -6,5 +6,15 @@ export interface PreferNodeProtocolRule {
    *
    * @see [prefer-node-protocol](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v46.0.0/docs/rules/prefer-node-protocol.md)
    */
-  'unicorn/prefer-node-protocol': Rule<[RuleLevel]>;
+  'unicorn/prefer-node-protocol': Rule<
+    [
+      RuleLevel,
+      {
+        /**
+         * @deprecated
+         */
+        checkRequire?: boolean;
+      },
+    ]
+  >;
 }
