diff --git a/src/rules/graphql-eslint/naming-convention.d.ts b/src/rules/graphql-eslint/naming-convention.d.ts
index be363af..a5c162b 100644
--- a/src/rules/graphql-eslint/naming-convention.d.ts
+++ b/src/rules/graphql-eslint/naming-convention.d.ts
@@ -83,8 +83,7 @@ export interface NamingConventionRule {
               VariableDefinition?: AsString | AsObject;
               allowLeadingUnderscore?: boolean;
               allowTrailingUnderscore?: boolean;
-              /**
-               */
+            } & {
               [k: string]: AsString | AsObject;
             },
           ]
