diff --git b/src/rules/graphql-eslint/naming-convention.d.ts a/src/rules/graphql-eslint/naming-convention.d.ts
index 60b228b..5e01373 100644
--- b/src/rules/graphql-eslint/naming-convention.d.ts
+++ a/src/rules/graphql-eslint/naming-convention.d.ts
@@ -78,8 +78,7 @@ export type NamingConventionOption =
         VariableDefinition?: AsString | AsObject;
         allowLeadingUnderscore?: boolean;
         allowTrailingUnderscore?: boolean;
-        /**
-         */
+      } & {
         [k: string]: AsString | AsObject;
       },
     ];
