diff --git a/src/rules/typescript-eslint/ban-ts-comment.d.ts b/src/rules/typescript-eslint/ban-ts-comment.d.ts
index 111f9c9..6015012 100644
--- a/src/rules/typescript-eslint/ban-ts-comment.d.ts
+++ b/src/rules/typescript-eslint/ban-ts-comment.d.ts
@@ -3,12 +3,42 @@ import type { RuleConfig } from '../rule-config';
 /**
  * Option.
  */
-export type BanTsCommentOption = any[];
+export interface BanTsCommentOption {
+  'ts-expect-error'?:
+    | boolean
+    | 'allow-with-description'
+    | {
+        descriptionFormat?: string;
+        [k: string]: any;
+      };
+  'ts-ignore'?:
+    | boolean
+    | 'allow-with-description'
+    | {
+        descriptionFormat?: string;
+        [k: string]: any;
+      };
+  'ts-nocheck'?:
+    | boolean
+    | 'allow-with-description'
+    | {
+        descriptionFormat?: string;
+        [k: string]: any;
+      };
+  'ts-check'?:
+    | boolean
+    | 'allow-with-description'
+    | {
+        descriptionFormat?: string;
+        [k: string]: any;
+      };
+  minimumDescriptionLength?: number;
+}
 
 /**
  * Options.
  */
-export type BanTsCommentOptions = BanTsCommentOption;
+export type BanTsCommentOptions = [BanTsCommentOption];
 
 /**
  * Disallow `@ts-<directive>` comments or require descriptions after directives.
