diff --git a/src/rules/typescript-eslint/lines-between-class-members.d.ts b/src/rules/typescript-eslint/lines-between-class-members.d.ts
index 46caad7..afdd87e 100644
--- a/src/rules/typescript-eslint/lines-between-class-members.d.ts
+++ b/src/rules/typescript-eslint/lines-between-class-members.d.ts
@@ -1,16 +1,23 @@
+import type {
+  LinesBetweenClassMembersConfig as BaseConfig,
+  LinesBetweenClassMembersOption,
+} from '../eslint/lines-between-class-members.d.ts';
 import type { RuleConfig } from '../rule-config.d.ts';
 
 /**
- * Option.
+ * Config.
  */
-export interface LinesBetweenClassMembersOption {
-  [k: string]: any;
+export interface LinesBetweenClassMembersConfig extends BaseConfig {
+  exceptAfterOverload?: boolean;
 }
 
 /**
  * Options.
  */
-export type LinesBetweenClassMembersOptions = LinesBetweenClassMembersOption;
+export type LinesBetweenClassMembersOptions = [
+  LinesBetweenClassMembersOption?,
+  LinesBetweenClassMembersConfig?,
+];
 
 /**
  * Require or disallow an empty line between class members.
