diff --git a/src/rules/typescript-eslint/padding-line-between-statements.d.ts b/src/rules/typescript-eslint/padding-line-between-statements.d.ts
index e0068e5..3efd25a 100644
--- a/src/rules/typescript-eslint/padding-line-between-statements.d.ts
+++ b/src/rules/typescript-eslint/padding-line-between-statements.d.ts
@@ -7,9 +7,10 @@ export interface PaddingLineBetweenStatementsRule {
    * @see [padding-line-between-statements](https://typescript-eslint.io/rules/padding-line-between-statements)
    */
   '@typescript-eslint/padding-line-between-statements': Rule<
-    [RuleLevel, ...('any' | 'never' | 'always')]
+    [RuleLevel, ...PaddingLineBetweenStatementsOption]
   >;
 }
+type PaddingType = 'any' | 'never' | 'always';
 type StatementType =
   | (
       | '*'
