diff --git a/src/rules/typescript-eslint/parameter-properties.d.ts b/src/rules/typescript-eslint/parameter-properties.d.ts
index 3f80f6f..9d95e8a 100644
--- a/src/rules/typescript-eslint/parameter-properties.d.ts
+++ b/src/rules/typescript-eslint/parameter-properties.d.ts
@@ -6,5 +6,32 @@ export interface ParameterPropertiesRule {
    *
    * @see [parameter-properties](https://typescript-eslint.io/rules/parameter-properties)
    */
-  '@typescript-eslint/parameter-properties': Rule<[RuleLevel, ...any[]]>;
+  '@typescript-eslint/parameter-properties': Rule<
+    [
+      RuleLevel,
+      {
+        allow?: [
+          (
+            | 'readonly'
+            | 'private'
+            | 'protected'
+            | 'public'
+            | 'private readonly'
+            | 'protected readonly'
+            | 'public readonly'
+          ),
+          ...(
+            | 'readonly'
+            | 'private'
+            | 'protected'
+            | 'public'
+            | 'private readonly'
+            | 'protected readonly'
+            | 'public readonly'
+          )[],
+        ];
+        prefer?: 'class-property' | 'parameter-property';
+      },
+    ]
+  >;
 }
