diff --git a/src/rules/react/jsx-no-constructed-context-values.d.ts b/src/rules/react/jsx-no-constructed-context-values.d.ts
index 410f060..e356693 100644
--- a/src/rules/react/jsx-no-constructed-context-values.d.ts
+++ b/src/rules/react/jsx-no-constructed-context-values.d.ts
@@ -1,17 +1,9 @@
 import type { RuleConfig } from '../rule-config';
 
-/**
- * Option.
- */
-export interface JsxNoConstructedContextValuesOption {
-  [k: string]: any;
-}
-
 /**
  * Options.
  */
-export type JsxNoConstructedContextValuesOptions =
-  JsxNoConstructedContextValuesOption;
+export type JsxNoConstructedContextValuesOptions = [];
 
 /**
  * Disallows JSX context provider values from taking values that will cause needless rerenders.
