diff --git a/src/rules/react/jsx-props-no-spreading.d.ts b/src/rules/react/jsx-props-no-spreading.d.ts
index c1e0069..ba1e1bc 100644
--- a/src/rules/react/jsx-props-no-spreading.d.ts
+++ b/src/rules/react/jsx-props-no-spreading.d.ts
@@ -8,8 +8,6 @@ export type JsxPropsNoSpreadingOption = {
   custom?: 'enforce' | 'ignore';
   exceptions?: string[];
   [k: string]: any;
-} & {
-  [k: string]: any;
 };
 
 /**
