diff --git a/src/rules/eslint/no-constructor-return.d.ts b/src/rules/eslint/no-constructor-return.d.ts
index 23c27dc..87369e1 100644
--- a/src/rules/eslint/no-constructor-return.d.ts
+++ b/src/rules/eslint/no-constructor-return.d.ts
@@ -6,12 +6,5 @@ export interface NoConstructorReturnRule {
    *
    * @see [no-constructor-return](https://eslint.org/docs/rules/no-constructor-return)
    */
-  'no-constructor-return': Rule<
-    [
-      RuleLevel,
-      ...{
-        [k: string]: any;
-      },
-    ]
-  >;
+  'no-constructor-return': Rule<[RuleLevel]>;
 }
