diff --git b/src/rules/eslint/no-constructor-return.d.ts a/src/rules/eslint/no-constructor-return.d.ts
index fedeca4..3e1fd03 100644
--- b/src/rules/eslint/no-constructor-return.d.ts
+++ a/src/rules/eslint/no-constructor-return.d.ts
@@ -1,16 +1,9 @@
 import type { RuleConfig } from '../rule-config.d.ts';
 
-/**
- * Option.
- */
-export interface NoConstructorReturnOption {
-  [k: string]: any;
-}
-
 /**
  * Options.
  */
-export type NoConstructorReturnOptions = NoConstructorReturnOption;
+export type NoConstructorReturnOptions = [];
 
 /**
  * Disallow returning value from constructor.
