diff --git a/src/rules/eslint/padding-line-between-statements.d.ts b/src/rules/eslint/padding-line-between-statements.d.ts
index aab5de2..2e0c62d 100644
--- a/src/rules/eslint/padding-line-between-statements.d.ts
+++ b/src/rules/eslint/padding-line-between-statements.d.ts
@@ -7,9 +7,10 @@ export interface PaddingLineBetweenStatementsRule {
    * @see [padding-line-between-statements](https://eslint.org/docs/rules/padding-line-between-statements)
    */
   'padding-line-between-statements': Rule<
-    [RuleLevel, ...('any' | 'never' | 'always')]
+    [RuleLevel, ...PaddingLineBetweenStatementsOption]
   >;
 }
+type PaddingType = 'any' | 'never' | 'always';
 type StatementType =
   | (
       | '*'
