diff --git a/src/rules/vitest/valid-title.d.ts b/src/rules/vitest/valid-title.d.ts
index 160be76..834ac9b 100644
--- a/src/rules/vitest/valid-title.d.ts
+++ b/src/rules/vitest/valid-title.d.ts
@@ -7,15 +7,7 @@ export interface ValidTitleOption {
   ignoreTypeOfDescribeName?: boolean;
   allowArguments?: boolean;
   disallowedWords?: string[];
-  /**
-   */
-  [k: string]:
-    | string
-    | [string]
-    | [string, string]
-    | {
-        [k: string]: string | [string] | [string, string];
-      };
+  [k: string]: any;
 }
 
 /**
